<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QToolButton" name="backButton">
       <property name="text">
        <string>←</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="forwardButton">
       <property name="text">
        <string>→</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="reloadButton">
       <property name="text">
        <string>↺</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="location"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="WebPage" name="webpage" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WebPage</class>
   <extends>QWidget</extends>
   <header location="global">webpage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
